pymc3/tests/test_step.py

```diff
@@ -982,7 +982,7 @@ def test_linalg(self, caplog):
             a = Normal("a", size=2, testval=floatX(np.zeros(2)))
             a = at.switch(a > 0, np.inf, a)
             b = at.slinalg.solve(floatX(np.eye(2)), a)
-            Normal("c", mu=b, size=2, testval=floatX(np.r_[0.0, 0.0]))
+            Normal("c", mu=b, shape=(2,), testval=floatX(np.r_[0.0, 0.0]))
```

Comment:
```suggestion
            Normal("c", mu=b, size=2, testval=floatX(np.r_[0.0, 0.0]))
```

Reply:
Same as with other examples: The way `size` was used before the `testval` shape does not match. Like with other examples, this was likely an unintended change from the original test when it was switched to `size`.